datahub/interaction/serializers.py

```diff
@@ -413,6 +482,65 @@ class Meta:
                     OperatorRule('contacts', lambda value: len(value) <= 1),
                     when=OperatorRule('is_event', bool),
                 ),
+                ValidationRule(
+                    'invalid_when_feature_flag_off',
+                    OperatorRule('were_countries_discussed', is_blank),
+                    OperatorRule('export_countries', is_blank),
+                    when=AndRule(
+                        IsObjectBeingCreated(),
+                        NotRule(IsFeatureFlagActive(INTERACTION_ADD_COUNTRIES)),
+                    ),
+                ),
+                ValidationRule(
+                    'invalid_for_investment',
+                    OperatorRule('were_countries_discussed', not_),
+                    when=EqualsRule('theme', Interaction.THEMES.investment),
+                ),
+                ValidationRule(
+                    'required',
+                    OperatorRule('were_countries_discussed', is_not_blank),
+                    when=AndRule(
+                        IsObjectBeingCreated(),
+                        IsFeatureFlagActive(INTERACTION_ADD_COUNTRIES),
+                        InRule(
+                            'theme',
+                            [Interaction.THEMES.export, Interaction.THEMES.other],
+                        ),
+                    ),
+                ),
+                ValidationRule(
+                    'required',
+                    OperatorRule('export_countries', is_not_blank),
+                    when=AndRule(
+                        OperatorRule('were_countries_discussed', bool),
+                        InRule(
+                            'theme',
+                            [Interaction.THEMES.export, Interaction.THEMES.other],
+                        ),
+                    ),
+                ),
+                ValidationRule(
+                    'invalid_when_no_countries_discussed',
+                    OperatorRule('export_countries', is_blank),
+                    when=AndRule(
+                        IsObjectBeingCreated(),
+                        IsFeatureFlagActive(INTERACTION_ADD_COUNTRIES),
+                        EqualsRule('were_countries_discussed', False),
```

Comment:
Should we make this:

```suggestion
                        OperatorRule('were_countries_discussed', not_),
```
?